static/admin/config.yml

```
@@ -23,11 +23,11 @@ collections:
         widget: "list"
         required: false
         fields:
-          - { label: "Title", name: "blogTitle", widget: "string"}
+          - { label: "Title", name: "blogTitle", widget: "string", pattern: ['.{5,60}', "Must have 5-60 characters"]}
           - { label: "Publish Date", name: "publishDate", widget: "datetime" }
-          - { label: "Label", name: "blogLabel", widget: "string" }
-          - { label: "Description", name: "blogDescription", widget: "string" }
-          - { label: "Body", name: "blogBody", widget: "markdown" }
+          - { label: "Label", name: "blogLabel", widget: "string", pattern: ['.{3,20}', "Must have 3-20 characters"]} }
+          - { label: "Description", name: "blogDescription", widget: "string", ['.{20,250}', "Must have 20-250 characters"]}
+          - { label: "Body", name: "blogBody", widget: "markdown", ['.{100,3000}', "Must have 100-3000 characters"] }
```

```suggestion
          - { label: "Description", name: "blogDescription", widget: "string", pattern: ['.{20,250}', "Must have 20-250 characters"]}
          - { label: "Body", name: "blogBody", widget: "markdown", pattern: ['.{100,3000}', "Must have 100-3000 characters"] }
```
add missing pattern keywords 😄

---

Fixed!